src/components/Layout/Header/ActionCenter/ActionCenter.tsx

```diff
@@ -89,6 +90,9 @@ export const ActionCenter = memo(() => {
           case ActionType.RfoxClaim: {
             return <RfoxClaimActionCard key={action.id} action={action} />
           }
+          case ActionType.TcyClaim: {
+            return <TcyClaimActionCard key={action.id} action={action} />
+          }
```

Comment:
q: now that we have multiple claim action cards, do we want a generic claim card that would take handlers or do the logic under the hood instead of duplicating both?

Reply:
Gave it a try locally and conceptually agree view-layer wise, but not sure about the discrimination of TCY vs. RFOX i.e one has asset *and* underlying asset, the other does not. Stay tuned though, giving it a bit more thought!

Reply:
Trusting you following this anyway, if you feel like it's not worth merging those too specially for TCY, happy to have it when we have another claim scenario that would fit more